bdb/column.go

```diff
@@ -29,6 +29,11 @@ type Column struct {
 	// tinyint(1) instead of tinyint
 	// Used for "tinyint-as-bool" flag
 	FullDBType string
+
+	// MS SQL only bits
+	// Used to indicate that the value
+	// for this column is auto generated by database on insert (i.e. - timestamp (old) or rowversion (new))
+	AutoGenerated bool
```

Comment:
How does this differ from a default value? To me these read the same. Or is it different in that a value can never be provided for it?